core/client/src/test/java/alluxio/client/file/FileSystemTestUtils.java

```diff
@@ -9,9 +9,11 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.client;
+package alluxio.client.file;
 
 import alluxio.AlluxioURI;
+import alluxio.client.ReadType;
+import alluxio.client.WriteType;
 import alluxio.client.file.FileOutStream;
```

Comment:
do you still need to import `alluxio.client.file.*` now that it has been moved to the package?